content/_data/en.json

```
@@ -96,7 +96,8 @@
       "auth0-authentication": "Auth0 Authentication",
       "okta-authentication": "Okta Authentication",
       "amazon-cognito-authentication": "Amazon Cognito Authentication",
-      "google-authentication": "Google Authentication"
+      "google-authentication": "Google Authentication",
+      "graphql": "GraphQL"
```

```suggestion
      "graphql": "Working with GraphQL"
```
@kevinold Since this is going under the "Testing Strategies" section, we shouldn't just title the page "GraphQL." It should be named "Working with GraphQL" or "Testing GraphQL." What's your preference?